doc/api/debugger.md

```diff
@@ -118,6 +118,9 @@ To begin watching an expression, type `watch('my_expression')`. The command
 functions body
 * `setBreakpoint('script.js', 1)`, `sb(...)`: Set breakpoint on first line of
 `script.js`
+* `setBreakpoint('script.js', 1, '<expression>')`, `sb(...)`: Set conditional
+breakpoint on first line of `script.js` that only breaks when the given
+`expression` evaluates to `true`.
```

Comment:
Maybe since the other arguments are examples, make `<expression>` an example? 

```suggestion
* `setBreakpoint('script.js', 1, 'num < 4')`, `sb(...)`: Set conditional
breakpoint on first line of `script.js` that only breaks when `num < 4`
evaluates to `true`
```

(Removed the period from the end for consistency with the other items in the list. Alternatively, add periods to all the other items.)

Reply:
Ah thanks for the quick review! I'll change that to be an example.